.github/workflows/run-cypress-tests.yml

```diff
@@ -0,0 +1,13 @@
+name: Cypress tests
+on: [pull_request]
+jobs:
+  cypress-run:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Run Cypress tests
+        uses: cypress-io/github-action@v2
```

Comment:
Similar question here with [using @v4 for cypress](https://github.com/cypress-io/github-action#explicit-version)